### ISSUE TITLE
Question whether custom particle packs and any other local visual changes are possible

### ISSUE THREAD

**Issue by andrew4043** [tetrio/issues#820]
### Checklist

- [x] I've **checked both [open and recently closed issues](https://github.com/tetrio/issues/issues?q=is%3Aissue)** to ensure this feature is not already requested.
- [X] I am using the **newest version** of TETR.IO.
- [X] I am certain this is a **feature request**, not a bug.
- [X] I've **thought this feature through** properly.

### Is your feature request related to a problem? Please describe.

Hi, a bit of a weird inquiry- Would it be possible for creators to make custom packs that customise other visual parts of TETR.IO ingame locally? Not just the blockskin, some other things I have in mind are: 
-The attack trails
-The board itself
-The score counter for 1v1s
-The spike number (able to customise its font, flashing pattern, color)
-The colors of the line clear particles(and by extension, the particles' shapes and any others); for example, light blue and all shades closely related to that

And any other ingame visuals that we're able to change locally? Understandable if it's not possible atm, or if I'd need an addon (which I know of that none of these exist and if an addon is needed then I'm okay with no for an answer as idk how to code)

tldr is the local visual part of tetrio moddable

### Describe the solution you'd like

Optimally, a step-by-step solution on how to change any of that
A list of which visuals can be changed, if possible, is also fine by me.

### Describe alternatives you've considered

n/a

**Comment by ZaptorZap** [comment]
I'd written out a very long response before something happened to my Github tab and it threw itself in the bin, though I did save a partial screenshot of the response since it was easily my longest Github thing, period. 
![image](https://user-images.githubusercontent.com/64891705/141238174-0535d386-9720-473b-aff2-ef2c66d9ff70.png)
Well, I'm not rewriting it so I'll just tldr answer your tldr question: it's very possible to change the particles as well, since those are stored in PNG files. You may find these PNG locations by simply having the devtools open, then filtering network requests to "img" before the game loads anything
![image](https://user-images.githubusercontent.com/64891705/141238361-bb01cdf4-103c-495e-bece-36e7ac483355.png)
Anyway, considering this wasn't a request I'll be closing this down now. I know it gets it's _reviews_ but consider [the Discord server's](https://l.tetr.io/discord) #tetrio channel, or, consider petitioning for osk to enable the Discussions tab in this repository for future questions.

**Comment by andrew4043** [comment]
leaving this here just for historical/documentation purposes, this feature is possible in the TETRIO PLUS mod starting in v0.22.1, released ~4AM UTC Nov 19
